Tests/TestingApps/PurchaseTesterSwiftUI/Shared/PurchaseTesterApp.swift

```diff
@@ -11,7 +11,18 @@ import RevenueCat
 
 @main
 struct PurchaseTesterApp: App {
-    @UIApplicationDelegateAdaptor(AppDelegate.self) var appDelegate
+
+    private let delegate = Delegate()
```

Comment:
Hmm I wonder, could we make this a local variable? Currently it's only used inside the `init` so we could just create it there I think.

Reply:
That wouldn't work. The delegate is `weak` so we do need to keep a reference around.

Reply:
Ahh you're totally right 👍